app/Http/Controllers/ProjectChangeRequestController.php

```
@@ -0,0 +1,60 @@
+namespace App\Http\Controllers;
+
+use App\Http\Requests\StoreProjectChangeRequest;
+use App\Http\Requests\UpdateProjectChangeRequest;
+use App\Http\Resources\ProjectChangeRequestResource;
+use App\Models\ProjectChangeRequest;
+
+class ProjectChangeRequestController extends Controller
+{
+    public function index()
+    {
+        $changeRequests = ProjectChangeRequest::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\ProjectChangeRequest' in method 'index'. (undefined)

(StaticAccess)